packages/compass/src/main/window-manager.js

```diff
@@ -319,6 +320,36 @@ function showAboutDialog() {
   });
 }
 
+function showLogFileDialog({ logFilePath }) {
+  dialog.showMessageBox({
+    type: 'info',
+    title: 'Log file for this session',
+    icon: COMPASS_ICON,
+    message: `The log file for this session can be found at ${logFilePath}`,
+    detail: 'Some tools may not be able to read the log file until Compass has exited.',
+    buttons: ['OK', 'Copy to clipboard', 'Open Folder', 'Extract']
```

Comment:
How about `Extract and open as .txt file` make `Extract` a bit more descriptive? I really like the functionality of it.

Reply:
Sure! I’ve shortened that a bit, because at least on Linux, these are displayed side-by-side, not stacked below each other :+1: